bug(lux-lua): luarocks package is broken

Installing the latest `lux-lua` version from luarocks appears to be broken due to a missing feature flag (vendored):

```shell
Run lx --lua-version 5.4 pack lux-lua
  lx --lua-version 5.4 pack lux-lua
  shell: /bin/bash -e {0}
  env:
    SITE_DIR: _site
    LUA_VERSION: 5.4
Error: 
   0: failed to build lux-lua: rust-mlua build failed: `cargo build` failed.
      status: exit status: 101
      stdout: 
      stderr: warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
      package:   /private/var/folders/dq/2t71w9pd3jx50vh3kp15_dt40000gn/T/.tmpDaStHR/lux-0.33.8/lux-cli/Cargo.toml
      workspace: /private/var/folders/dq/2t71w9pd3jx50vh3kp15_dt40000gn/T/.tmpDaStHR/lux-0.33.8/Cargo.toml
      error: the package 'lux-lua' does not contain this feature: vendored
      help: packages with the missing feature: lux-lib, lux-cli, lux-definitions
      

Location:
   lux-cli/src/pack.rs:112

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Error: Process completed with exit code 1.

```

--- a/lux.toml
+++ b/lux.toml
@@ -16,7 +16,7 @@ dev = "git+https://github.com/lumen-oss/lux.git"
 [build]
 type = "rust-mlua"
 cargo_extra_args = ["--package", "lux-lua", "--locked"]
-features = ["vendored", "module"]
+features = ["vendored-module"]
 
 [build.modules]
 lux = "lux_lua"

